### ISSUE TITLE
Should people be allowed to zoom into the page?

### ISSUE THREAD

**Issue by anevins12** [FSWebDevBook/speedscore#10]
Should people be allowed to zoom into the page using the native zoom controls on their devices or user agents?

Zooming can be a useful tool for people in general, but may be more critical for people with moderately low vision when using apps and websites.

**Comment by chundhau** [comment]
@anevins12  Yes, I believe zooming in is reasonable. How can I accommodate this in the HTML, CSS, and JavaScript?

**Comment by anevins12** [comment]
Zooming functionality should already exist from the browser or operating system via the `ctrl` (windows)/ `cmd` (Apple) keys combined with the `+` key. If that's not working (I think it's not) then I assume that there must be some script disabling that on the page. Or perhaps it's a meta tag somewhere in the `<head>`. I haven't had the chance to look into this a second time
